src/main/webapp/resources/js/components/samples/components/EditMetadata.jsx

```diff
@@ -0,0 +1,175 @@
+import {
+  Form,
+  Input,
+  Modal,
+  notification,
+  Popover,
+  Select,
+  Typography,
+} from "antd";
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useUpdateSampleMetadataMutation } from "../../../apis/samples/samples";
+import { useResetFormOnCloseModal } from "../../../hooks";
+import { useMetadataRoles } from "../../../contexts/metadata-roles-context";
+import { editSampleMetadata, setEditSampleMetadata } from "../sampleSlice";
+import { IconWarningOutlined } from "../../icons/Icons";
+import { grey6 } from "../../../styles/colors";
+import { WarningAlert } from "../../alerts";
+
+const { Title } = Typography;
+/**
+ * Function to render Update Metadata Field modal
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function EditMetadata() {
+  const dispatch = useDispatch();
+  const [updateSampleMetadata] = useUpdateSampleMetadataMutation();
+  const { roles: metadataRoles } = useMetadataRoles();
+  const {
+    editModalVisible,
+    field,
+    fieldId,
+    entry,
+    entryId,
+    restriction,
+    sample,
+    projectId,
+  } = useSelector((state) => state.sampleReducer);
+
+  const [form] = Form.useForm();
+
+  useResetFormOnCloseModal({
+    form,
+    visible: editModalVisible,
+  });
+
+  const updateMetadata = () => {
+    form.validateFields().then((values) => {
+      updateSampleMetadata({
+        sampleId: sample.identifier,
+        projectId,
+        metadataFieldId: fieldId,
+        metadataField: values.metadata_field_name,
+        metadataEntryId: entryId,
+        metadataEntry: values.metadata_field_value,
+        metadataRestriction: values.metadata_field_permission,
+      })
+        .then((response) => {
+          if (response.error) {
+            notification.error({ message: response.error.data.error });
```

Comment:
Does this ever get hit?  Should it not get caught on line 77?

Reply:
You are correct. Looks like I forgot to remove that once I moved the catching of it into the then statement. Removed in 54adb73843ab4d44d03e431f3dc155e70ea2b16d